secure_message/resources/messages.py

```diff
@@ -34,11 +34,16 @@ def post(self):
 
         if 'msg_id' in post_data:
             raise BadRequest(description="Message can not include msg_id")
+        if post_data.get('thread_id') in post_data:
```

Comment:
You can drop the `in post_data`.

`post_data.get('thread_id')` will return `None` or a `<thread_id>`, so this statement is actually looking for a key that matches `<thread_id>` in post_data.